control/subscription_group.go

```diff
@@ -51,13 +51,12 @@ type ManagesSubscriptionGroups interface {
 	Add(id string, requested []core.RequestedMetric,
 		configTree *cdata.ConfigDataTree,
 		plugins []core.SubscribedPlugin) []serror.SnapError
-	Get(id string) ([]core.Metric, []serror.SnapError, error)
+	Get(id string) (map[string]metricTypes, []serror.SnapError, error)
```

Comment:
I tried to use metricType as a first, but it has metricType.Policy(), metricTypes does not have to, so only metricTypes can be used as input for plugin.CollectMetrics() - is that make sense?